sensors:
  steering_front:
    rate: 200
    dead_time: 0.005
    error:
      mean: 0.0
      sigma: 0.00001
  steering_rear:
    rate: 200
    dead_time: 0.005
    error:
      mean: 0.0
      sigma: 0.00001
  wheelspeeds:
    rate: 200
    dead_time: 0.005
    error:
      mean: 0.0
      sigma: 0.1
  torques:
    rate: 200
    dead_time: 0.005
    error:
      mean: 0.0
      sigma: 0.01
  voltage_ts:
    rate: 200
    dead_time: 0.005
    error:
      mean: 0.0
      sigma: 0.00001
  current_ts:
    rate: 200
    dead_time: 0.005
    error:
      mean: 0.0
      sigma: 0.00001
  imus:
    - imu:
      name: "cog_imu"
      frame: "cog_static"
      pose:
        position: [0.0,0.0,0.0]
        orientation: [0.0,0.0,0.0]
      rate: 200
      dead_time: 0.005
      error_acc:
        mean: 0.0
        sigma: 0.05
      error_rot:
        mean: 0.0
        sigma: 0.001
  gnssSensors:
  - gnss:
    name: "gnss_front"
    frame: "gnss_front"
    pose:
      position: [0.66,0.0,0.64]
      orientation: [0.0,0.0,0.0]
    rate: 20
    delay: 
      mean: 0.05
    features:
      velocity: true
      orientation: 0 # 0 : no orientaiton, 1 : heading only, 2 : full orientation
    noise:
      position: # enu plane
        mean: [0.0, 0.0, 0.0]
        sigma: [0.03, 0.03, 1.3]
      velocity: # enu plane
        mean: [0.0, 0.0, 0.0]
        sigma: [0.02, 0.02, 0.02]
      orientation: # enu plane
        mean: [0.0, 0.0, 0.0]
        sigma: [0.02, 0.02, 0.02]